tests/test_recover.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from brownie import Wei, reverts, interface
+from utils.config import (
+    lido_dao_agent_address,
+    ldo_token_address,
+    lido_dao_voting_address,
+    lido_dao_token_manager_address
+)
+from utils.evm_script import encode_call_script
+from utils.voting import create_vote
+
+
```

Comment:
I propose to add a test for a non-conformant ERC-20 token (USDT, for example). This one would leverage our low-level `safeTransfer` call.